### ISSUE TITLE
Doesn't compile on MacOs

### ISSUE THREAD

**Issue by Emoun** [t-crest/patmos-llvm#27]
Compilation fails on MacOs. When I run it, I get an error about various labels being undefined for x86_64:

This is not necessarily an issue with the code, as it also doesn't compile as far back as commit 0c175299f79999f2b0f76d7f72f60d23a8ba2886. Therefore, this might have something to do with the age of the project, and could be solved by updating to a newer version of LLVM.

**Comment by schoeberl** [comment]
This issue is known, as I reported it long time ago. But I don't find the issue now. My LLVM on macOS is resent.

**Comment by Emoun** [comment]
Yes, I just wanted to have an open issue on it to be able to track it.

**Comment by schoeberl** [comment]
Related question (not this issue): there was a precompiled version for Mac available (with ```build.sh -q llvm2```), but it gave some link errors:
```
<unknown>:0: error: symbol '.LBB305_1' can not be undefined in a subtraction expression
<unknown>:0: error: symbol '.LBB579_1' can not be undefined in a subtraction expression
```

Is there any progress in this direction? Would really love to see this working on macOS, as I'm currently trying to get lingua franca running again on Patmos here at UCB.

**Comment by Emoun** [comment]
@schoeberl I'm currently looking into getting the new toolchain updated for use with the build script. So, hopefully by next week I'm at a place where others can test.
